casper/src/test/scala/coop/rchain/casper/engine/RunningMaintainRequestedBlocksSpec.scala

```diff
@@ -35,7 +35,7 @@ class RunningMaintainRequestedBlocksSpec extends FunSpec with BeforeAndAfterEach
     describe("maintainRequestedBlocks, for every block that was requested") {
       describe("if block request is still within a timeout") {
         it("should keep the request not touch") {
-          val requested                = Requested(timestamp = notTimedOut)
+          val requested                = Requested(timestamp = notTimedOut, received = false)
```

Comment:
On many places now received doesn't have to be set, default is false.

Reply:
https://github.com/rchain/rchain/pull/2902/commits/372522ce494a5493354b7a3505316204e77eb715